QuackNBOOM/QuackNBOOM/Duck.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Duck.swift
+//  QuackNBOOM
+//
+//  Created by Ngo Tuyetnhi N. and Benoit Neriah R. on 3/4/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+// Duck inherits from GameObject with added functionality specific to the falling duck.
+
+import Foundation
+import SpriteKit
+
+class Duck: GameObject {
+    //boolean if duck is dead or not
+    let isDead = false; //ducks start as 'not dead'
+    let startPos = CGPoint(x: 1000, y: 1750)
+    //speed of the duck, make faster
+    let vel = CGFloat(200)
+    
+    // Sets the sprite's image
+    required init(imagePath: String){
+        super.init(imagePath: imagePath)
+        zPos = 1.0
+        // Proper scale for the ducks
+        scale = CGSize(width: 0.75, height: 0.75)
+    }
+    
+    // Update is called every frame and holds the main functionality of the duck
+    override func update(_deltaTime: TimeInterval) {
```

Comment:
Just put a space between the "_" and "deltaTime" if you are wishing to omit the label for the argument when you use it.

Swift actually lets you rename the labels to whatever you wish, so long as the function signature is the same in respect to variable type etc.

Reply:
**This requested change has been successfully implemented**
Made all the necessary changes!